src/services/car/inventoryData.ts

```diff
@@ -1,49 +1,55 @@
 import { fetchPath, Service } from "../../base"
 
 export const fetchBodyTypes = (): Promise<string[]> => {
-  return fetchPath(Service.CAR, "inventory/bodytypes")
+  return fetchPath({ service: Service.CAR, path: "inventory/bodytypes" })
 }
 
 export const fetchColorGroups = (): Promise<string[]> => {
-  return fetchPath(Service.CAR, "inventory/colorgroups")
+  return fetchPath({ service: Service.CAR, path: "inventory/colorgroups" })
 }
 
 export const fetchColors = (): Promise<string[]> => {
-  return fetchPath(Service.CAR, "inventory/colors")
+  return fetchPath({ service: Service.CAR, path: "inventory/colors" })
 }
 
 export const fetchConditionTypes = (): Promise<string[]> => {
-  return fetchPath(Service.CAR, "inventory/conditiontypes")
+  return fetchPath({ service: Service.CAR, path: "inventory/conditiontypes" })
 }
 
 export const fetchDoors = (): Promise<number[]> => {
-  return fetchPath(Service.CAR, "inventory/doors")
+  return fetchPath({ service: Service.CAR, path: "inventory/doors" })
 }
 
 export const fetchDriveTypes = (): Promise<string[]> => {
-  return fetchPath(Service.CAR, "inventory/drivetypes")
+  return fetchPath({ service: Service.CAR, path: "inventory/drivetypes" })
 }
 
 export const fetchFuelTypeGroups = (): Promise<string[]> => {
-  return fetchPath(Service.CAR, "inventory/fueltypegroups")
+  return fetchPath({ service: Service.CAR, path: "inventory/fueltypegroups" })
 }
 
 export const fetchFuelTypes = (): Promise<string[]> => {
-  return fetchPath(Service.CAR, "inventory/fueltypes")
+  return fetchPath({ service: Service.CAR, path: "inventory/fueltypes" })
 }
 
 export const fetchMinFirstRegistrationYear = (): Promise<number> => {
-  return fetchPath(Service.CAR, "inventory/min-first-registration-year")
+  return fetchPath({
+    service: Service.CAR,
+    path: "inventory/min-first-registration-year",
+  })
 }
 
 export const fetchOptions = (): Promise<string[]> => {
-  return fetchPath(Service.CAR, "inventory/options")
+  return fetchPath({ service: Service.CAR, path: "inventory/options" })
 }
 
 export const fetchSeats = (): Promise<number[]> => {
-  return fetchPath(Service.CAR, "inventory/seats")
+  return fetchPath({ service: Service.CAR, path: "inventory/seats" })
```

Comment:
this diff illustrates pretty well how much more code we overall generate to achieve the same as with regular attributes :)